src/_resources/case-study-carb-mobility-loyalty-rewards-market-sounding.md

```diff
@@ -0,0 +1,8 @@
+---
+date: "2023-11-01T17:00:00-07:00"
+title: "Cal-ITP / CARB Market Sounding: Mobility Rewards Strategy & Demonstration Project"
+asset: CARB.Mobility.Loyalty.and.Rewards.Market.Sounding.Background.pdf
+category: Case studies
```

Comment:
This all looks good to me.

Just one thought @mrose914 maybe we need a different category. A few of the recent ones don't really fit into "Case study" -- more like after-action reports or something.